javascript/javascript1/week3/smartphoneapp.js

```diff
@@ -0,0 +1,80 @@
+const activities = []
+function addActivity(activity, duration){
+    var today = new Date();
```

Comment:
Try not to use `var`. Use `const` or `let`. Think what would be more appropriate here.

Reply:
done